stages/org.osbuild.sgdisk

```diff
@@ -30,6 +30,11 @@ SCHEMA_2 = r"""
       "description": "UUID for the disk image's partition table",
       "type": "string"
     },
+    "no_quote_partition_names": {
```

Comment:
Negative booleans like this are always a little confusing (setting to `true` to disable) and the name is a bit too long and awkward.  Could we rename it to something terse?
`quote_names` (with default set to `True`) was the first thing that came to mind.

But thinking about this more, I wonder if we can go a different way and add a new property to the `partition` object called something like `raw_name`, changing the current `name` description to read `The partition name. This name will be quoted. Use "raw_name" instead`.  Then we could either fail if both are provided or prioritise the raw name over the plain one.

Reply:
I agree that the name is not ideal and I expected others to provide better suggestions 😅 

However, I make the default value `False` intentionally, so that we can then `omitempty` the default value for `bool` in composer. I can make it a pointer in composer, so that we can omit the empty value and still treat `False` as not being an empty value.

As to your second suggestion, I find is acceptable, but `raw_name` seems confusing to me as it has no meaning in `sgdisk` AFAICT. But in principle it would be probably a better approach.

Reply:
I implemented changes. However, I kept the name `quote_partition_name` to keep the relationship with the `name` property of a partition in the stage options.